src/modules/bigquery.js

```diff
@@ -110,6 +111,26 @@ export default class BigQuery {
 
   buildOrderBy = (property, direction) => (property ? `ORDER BY ${property} ${direction}` : '');
 
+  getTotalRows = (projectId, query, token) => {
+    const url = `https://bigquery.googleapis.com/bigquery/v2/projects/${projectId}/queries`;
+
+    return axios
+      .post(
+        url,
+        {
+          query,
+          useLegacySql: false,
+        },
+        {
+          headers: {
+            'Content-Type': 'application/json',
+            Authorization: `Bearer ${token}`,
+          },
+        },
+      )
+      .then((response) => response.data.rows[0].f[0].v);
+  };
```

Comment:
i think if we're putting this in the store, we should probably move this call to `repository.js` to be consistent